README.md

```diff
@@ -9,3 +9,57 @@ This repository contains code to download and analyze data from the [Mini-Bird B
 ## Notes on the analysis data files
 
 * Prio
+
+
+### Instructions for updating the data
```

Comment:
General comment on this file - make sure that the markdown formats correctly. For example, you can see how GitHub flavored markdown renders the file by clicking `view file`. Here's the rendering of the latest commit: 
https://github.com/nc-minibbs/mbbs/blob/33b11e198827b5112a1b2d44bedd2e5c640f0b57/README.md.

I would put each step in a section header, as in 

```md
#### Step 1: Checking and QC'ing ebird data
```